pkg/tests/integration/test_salt_grains.py

```diff
@@ -39,3 +39,12 @@ def test_grains_setval_key_val(salt_cli, salt_minion):
     ret = salt_cli.run("grains.setval", "key", "val", minion_tgt=salt_minion.id)
     assert ret.data, ret
     assert "key" in ret.data
+
+
+def test_grains_package_onedir(salt_cli, salt_minion):
+    """
+    Test that the package grain returns onedir
+    """
+    ret = salt_cli.run("grains.get", "package", minion_tgt=salt_minion.id)
+    assert ret.data == "onedir"
```

Comment:
```suggestion
    if os.environ.get("ONEDIR_TESTRUN", "0") == "1":
        assert ret.data == "onedir"
    else:
        assert ret.data != "onedir"
```
It's always `onedir` on CI, but it might not be on our laptops.

Reply:
These are system package tests. It should always be onedir. In what scenario would somone be running the package tests and its not a onedir package?